api/types/semaphore.go

```diff
@@ -51,6 +51,10 @@ const SemaphoreKindAccessMonitoringLimiter = "access_monitoring_limiter"
 // session recordings backend.
 const SemaphoreKindUploadCompleter = "upload_completer"
 
+// SemaphoreKindAccessListReminderLimiter is the semaphore kind used by
+// the periodic check which creates access list reminder notifications.
+const SemaphoreKindAccessListReminderLimiter = "access_monitoring_limiter"
```

Comment:
Why is it called `access_monitoring_limiter`? Access Monitoring is a Teleport feature that's not related to access lists or notifications in any way. Maybe just a copy-paste?